src/views/primary-auth/PrimaryAuthForm.js

```diff
@@ -191,6 +191,22 @@ define([
       if (this.settings.get('features.trackTypingPattern')) {
         TypingUtil.track('okta-signin-username');
       }
-    }
+    },
+
+    // TODO: support a11y attrs in Courage - OKTA-279025
+    render: function () {
+      Okta.Form.prototype.render.call(this);
+
+      this.$el.attr('novalidate', 'novalidate');
+
+      [
+        this.getUsernameField().inputId, 
+        this.getPasswordField().inputId
+      ].forEach(inputId => {
```

Comment:
This approach (enumerating some inputs and finding them by ID) will work. However, you could also encapsulate this logic within the input definition by putting a `render()` method directly on the input defined in `getUsernameField` and `getPasswordField`. See https://github.com/okta/okta-signin-widget/blob/master/src/EnrollCallAndSmsController.js#L209 for an example. Doing it this way you won't need to find the input by ID, `this.$` will already be scoped to the individual input. This would help keep the logic for the input all together in one place.

Reply:
done